Lib/test/test_sqlite3/test_userfunctions.py

```diff
@@ -223,6 +224,14 @@ def test_func_error_on_create(self):
         with self.assertRaises(sqlite.OperationalError):
             self.con.create_function("bla", -100, lambda x: 2*x)
 
+    def test_func_too_many_args(self):
+        category = sqlite.SQLITE_LIMIT_FUNCTION_ARG
+        msg = "too many arguments on function"
+        with cx_limit(self.con, category=category, limit=1):
+            self.con.create_function("addint", 2, lambda x, y: x+y)
+            with self.assertRaisesRegex(sqlite.OperationalError, msg):
+                self.con.execute("select addint(1, 2)");
```

Comment:
```suggestion
            self.con.create_function("sqr", 1, lambda x: x**2)
            self.con.execute("select sqr(2)")
            self.con.create_function("addint", 2, lambda x, y: x+y)
            with self.assertRaisesRegex(sqlite.OperationalError, msg):
                self.con.execute("select addint(1, 2)")
```

Reply:
See 9521fc1754946fe6d9c2d5d190c63ff42d74eced and c53ffd7841a0dd63c6e09244131be0aaeab6e028